engine/schema/src/com/cloud/upgrade/dao/Upgrade452to460.java

```diff
@@ -66,10 +66,25 @@ public boolean supportsRollingUpgrade() {
     @Override
     public void performDataMigration(final Connection conn) {
         updateSystemVmTemplates(conn);
+        alterAddColumnToCloudUsage(conn);
         updateVMInstanceUserId(conn);
         addIndexForVMInstance(conn);
     }
 
+    public void alterAddColumnToCloudUsage(final Connection conn) {
```

Comment:
@abhinandanprateek the only fix needed now is to move this db migration from 452to460, to Upgrade460to471 class since we're going to merge on 4.7/master.